src/NUnitFramework/framework/Constraints/DelayedConstraint.cs

```diff
@@ -75,6 +75,18 @@ public override string Description
         }
 
         /// <summary>
+        /// Converts delayInMilliseconds input from minutes to milliseconds
+        /// </summary>
+        public DelayedConstraint Minutes
+        {
+            get
+            {
+                delayInMilliseconds = (int)TimeSpan.FromMinutes(delayInMilliseconds).TotalMilliseconds;
```

Comment:
Suppose I use `After(60).Minutes.Minutes`?

Take a look at how EqualConstraint handles Minutes as well as the other time units. There is a tolerance member with a numeric value and a unit. The unit gets set by the modifier property. If more than one is used, we use the last one seen. I could have generated a runtime error, but that seems unnecessary and a compile-time error is much more complicated.

Additionally, if we are having minutes, let's have Milliseconds and Seconds as well. You can initialize the units to milliseconds. Longer than minutes, seems silly in this context.